tests/js/handler-caseholder.js

```diff
@@ -0,0 +1,61 @@
+// Test `gpii.schema.middleware` and its subcomponents.
+//
+"use strict";
+var fluid        = fluid || require("infusion");
+var gpii         = fluid.registerNamespace("gpii");
+var jqUnit       = require("jqUnit");
+
+require("../../node_modules/gpii-express/tests/js/lib/test-helpers");
```

Comment:
I followed your advice from a previous pull request and replaced this with a call to `require("gpii-express");` followed by `gpii.express.loadTestingSupport()`.